doc/api/punycode.md

```diff
@@ -2,11 +2,14 @@
 
     Stability: 2 - Stable
 
-[Punycode.js][] is bundled with Node.js v0.6.2+. Use `require('punycode')` to
+[Punycode.js][] is bundled with Node.js v0.6.1+. Use `require('punycode')` to
```

Comment:
So, this should be v0.5.1.  The first punycode.js was https://github.com/bnoordhuis/punycode.js and was later replaced with https://github.com/bestiejs/punycode.js.